include/aws/common/byte_buf.h

```diff
@@ -361,6 +395,13 @@ uint64_t aws_hash_array_ignore_case(const void *array, size_t len);
 AWS_COMMON_API
 uint64_t aws_hash_byte_cursor_ptr_ignore_case(const void *item);
 
+/**
+ * Returns the lookup table used internally to do case-insensitive comparisons.  Allows it to be
+ * used in the lookup version of byte_buf append.
+ */
+AWS_COMMON_API
+const uint8_t *aws_lookup_table_to_lower_get(void);
```

Comment:
[trivial] "Returns table where uppercase ascii characters are replaced by lowercase. Can be used with..."
instead of explaining how its used internally

also, I'd be fine with leaving "_get" off the end of the name. but that's just, like, my opinion man